src/web/lib/middleware.js

```diff
@@ -1,4 +1,5 @@
 import { actions, selectors, themeChangeActions } from "../../lib/store";
+export let storedImages = new Map();
```

Comment:
To emphasize that this is not persistent, but a memory-only storage area, rename it to `temporaryImageStore`.